.eslintrc.json

```diff
@@ -0,0 +1,42 @@
+{
+    "env": {
+      "browser": true,
+      "commonjs": true,
+      "es6": true,
+      "node": true,
+      "mocha": true
+    },
+    "extends": ["google"],
+    "globals": {
+      "Atomics": "readonly",
+      "SharedArrayBuffer": "readonly"
+    },
+    "ignorePatterns": [
+      "**/node_modules",
+      "**/sitespeed-result",
+      "CODEOWNERS",
+      "*.yml",
+      "*.md",
+      "package-lock.json"
+    ],
+    "overrides": [
+      {
+        "files": ["dist/*.js", "src/*.js", "demo/demo.js"],
```

Comment:
dist is toch gegenereerde code? dus die zou ik niet verwachten dat moet gelint worden?

Reply:
Kan @cambiph misschien excluden in de vl-ui-util?

Reply:
Besproken met @cambiph en dist exclude zal toegepast worden op vl-ui-util niveau.